specification/datasets/cost_and_usage/columns/allocatedtags.md

```diff
@@ -0,0 +1,62 @@
+# Allocated Tags
```

Comment:
I'm not going to block on this, but I do not think we should add this column. This column requires practitioners to manage 2 sets of tags. Tags are difficult enough with 1 column. This will make a painful process even more painful with unnecessary complexity. I would rather use Tags for the allocated tags.